INFO:     127.0.0.1:62361 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:62361 - "GET /orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:62361 - "GET /static/js/pending_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:62361 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:62367 - "GET /static/js/components/receive_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:62373 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:62372 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:62369 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:62369 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:62372 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:62372 - "GET /orders/api/orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:62438 - "GET /orders/api/orders/pending_orders?requester=Aaron HTTP/1.1" 200 OK
INFO:     127.0.0.1:62624 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:62881 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63134 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63667 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63923 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:64176 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:64421 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:64686 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:64943 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:65207 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:65311 - "GET /maintenance HTTP/1.1" 200 OK
INFO:     127.0.0.1:65311 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:65313 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:65311 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.2/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 322, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.2/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 123, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:65316 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:65318 - "GET /lookups/users HTTP/1.1" 200 OK
INFO:     127.0.0.1:65319 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:65321 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:65321 - "GET /orders/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:65321 - "GET /static/js/audit_trail.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:65321 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:65318 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:65319 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:65316 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:65318 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:65316 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:65318 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.2/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 322, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.2/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 123, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:65316 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:65316 - "GET /orders/api/items_for_order/55 HTTP/1.1" 200 OK
INFO:     127.0.0.1:65316 - "GET /orders/attachments/55 HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:65316 - "GET /orders/attachments/55 HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:65449 - "GET /orders/api/items_for_order/22 HTTP/1.1" 200 OK
INFO:     127.0.0.1:65479 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:49232 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:49369 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:49561 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:49884 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:50231 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:50498 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:50762 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:51059 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:51429 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:51727 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:52001 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:52382 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:52736 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:52992 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     Will watch for changes in these directories: ['/Users/stevencohen/Projects/universal_recycling/orders_project/backend']
INFO:     Uvicorn running on http://0.0.0.0:8004 (Press CTRL+C to quit)
INFO:     Started reloader process [17079] using StatReload
INFO:     Started server process [17150]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:53513 - "GET /orders/new HTTP/1.1" 200 OK
INFO:     127.0.0.1:53513 - "GET /static/js/new_order_main.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:53513 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:53513 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:53513 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:53513 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:53513 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:53513 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:53516 - "GET /orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:53516 - "GET /static/js/pending_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:53516 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:53518 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:53520 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:53520 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:53518 - "GET /static/js/components/receive_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:53518 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:53520 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:53520 - "GET /orders/api/orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:53523 - "GET /orders/api/orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:53524 - "GET /orders/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:53524 - "GET /static/js/received_orders.js HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:53524 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:53524 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:53524 - "GET /orders/api/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:53537 - "GET /orders/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53537 - "GET /static/js/audit_trail.js HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:53537 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:53539 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:53537 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:53539 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53541 - "GET /orders/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53541 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:53543 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:53543 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53547 - "GET /orders/maintenance HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:53547 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:53549 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53549 - "GET /maintenance HTTP/1.1" 200 OK
INFO:     127.0.0.1:53549 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:53551 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:53555 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:53556 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:53557 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:53559 - "GET /lookups/users HTTP/1.1" 200 OK
INFO:     127.0.0.1:53555 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:53572 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53586 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53588 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53604 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53611 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53616 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53629 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53634 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53636 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53655 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53659 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53663 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53669 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53687 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53718 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53723 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53725 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53738 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53740 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53744 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53747 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53775 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53818 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53835 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53853 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53872 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53887 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53904 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53917 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53944 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53958 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:53973 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54017 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54035 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54066 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54082 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54108 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54113 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54126 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54153 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54167 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54182 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54197 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54226 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54246 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54274 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54287 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54304 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54319 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54335 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54364 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54382 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54412 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54439 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54467 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54483 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54496 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54525 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54539 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54556 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54577 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54600 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54617 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54633 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54666 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54682 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54697 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54704 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54706 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54729 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54744 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54748 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54753 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54775 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54796 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54824 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54841 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54866 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54906 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54921 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54946 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54961 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54976 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:54992 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55012 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55033 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55048 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55065 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55078 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55096 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55121 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55136 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55154 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55173 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55190 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55220 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55233 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55250 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55265 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55280 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55308 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55325 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55344 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55361 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55385 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55414 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55428 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55444 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55457 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55484 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55500 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55517 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55531 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55552 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55571 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55593 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55610 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55627 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55640 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55655 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55669 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55686 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55701 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55717 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55738 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55755 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55768 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55787 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55801 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55860 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55880 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55896 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55917 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55925 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55954 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55971 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:55984 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56000 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56002 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56007 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56010 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56054 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56085 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56150 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56186 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56203 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56227 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56244 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56281 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56354 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56368 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56395 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56409 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56436 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56470 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56500 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56524 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56540 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56560 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56585 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56588 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56614 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56619 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56649 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56664 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56672 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56685 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56691 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56704 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56709 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56723 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56739 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56742 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56751 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56774 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56781 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56795 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56829 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56842 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56858 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56872 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56892 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56907 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56926 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56945 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56954 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56967 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:56987 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57005 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57040 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57054 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57069 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57084 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57096 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57132 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57139 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57168 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57188 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57206 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57232 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57259 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57277 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57295 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57311 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57335 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57363 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57389 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57423 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57448 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
WARNING:  StatReload detected changes in 'backend/endpoints/auth.py'. Reloading...
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [17150]
INFO:     Started server process [22273]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:57471 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57497 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57525 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
WARNING:  StatReload detected changes in 'backend/main.py'. Reloading...
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [22273]
INFO:     Started server process [22352]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:57533 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     Will watch for changes in these directories: ['/Users/stevencohen/Projects/universal_recycling/orders_project/backend']
INFO:     Uvicorn running on http://0.0.0.0:8004 (Press CTRL+C to quit)
INFO:     Started reloader process [22568] using StatReload
INFO:     Started server process [22641]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:57559 - "GET /orders/new HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /static/js/new_order_main.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /static/js/pending_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57559 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57561 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57566 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57567 - "GET /static/js/components/receive_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57567 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /orders/api/orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /orders/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /static/js/received_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57581 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57585 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /orders/api/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /orders/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /static/js/audit_trail.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57585 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57581 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57581 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:57563 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:57581 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57581 - "GET /maintenance HTTP/1.1" 200 OK
INFO:     127.0.0.1:57581 - "GET /lookups/users HTTP/1.1" 200 OK
INFO:     127.0.0.1:57583 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:57585 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:57581 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:57587 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:57589 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:57591 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:57617 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57638 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57638 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:57638 - "GET /static/css/styles.css HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:57642 - "GET /static/js/login.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57642 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:57658 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57673 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57688 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57709 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57735 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57757 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57771 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     Will watch for changes in these directories: ['/Users/stevencohen/Projects/universal_recycling/orders_project/backend']
INFO:     Uvicorn running on http://0.0.0.0:8004 (Press CTRL+C to quit)
INFO:     Started reloader process [22988] using StatReload
INFO:     Started server process [23061]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:57791 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:57791 - "GET /static/css/styles.css HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:57793 - "GET /static/js/login.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:57793 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:57798 - "POST /login HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 93, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 144, in simple_response
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<3 lines>...
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/backend/endpoints/auth.py", line 15, in login
    cursor.execute("SELECT * FROM users WHERE username=? AND password=?", (username, password))
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: password
INFO:     127.0.0.1:57800 - "POST /login HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 93, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 144, in simple_response
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<3 lines>...
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/backend/endpoints/auth.py", line 15, in login
    cursor.execute("SELECT * FROM users WHERE username=? AND password=?", (username, password))
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: password
INFO:     127.0.0.1:57802 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:57802 - "GET /static/css/styles.css HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:58045 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
WARNING:  StatReload detected changes in 'backend/endpoints/auth.py'. Reloading...
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [23061]
INFO:     Started server process [23119]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Will watch for changes in these directories: ['/Users/stevencohen/Projects/universal_recycling/orders_project/backend']
INFO:     Uvicorn running on http://0.0.0.0:8004 (Press CTRL+C to quit)
INFO:     Started reloader process [23317] using StatReload
INFO:     Started server process [23388]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:58312 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:58312 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:58312 - "GET /static/css/styles.css HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:58337 - "GET /static/js/login.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:58337 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:58577 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:58850 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:59101 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:59365 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:59608 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:59878 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:60133 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:60377 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:60641 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:60903 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:61174 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:61430 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:61670 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:61931 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:62185 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     Will watch for changes in these directories: ['/Users/stevencohen/Projects/universal_recycling/orders_project/backend']
INFO:     Uvicorn running on http://0.0.0.0:8004 (Press CTRL+C to quit)
INFO:     Started reloader process [24554] using StatReload
INFO:     Started server process [24630]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     127.0.0.1:62434 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:62688 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:62933 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63202 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63251 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:63251 - "GET /static/css/style.css HTTP/1.1" 200 OK
INFO:     127.0.0.1:63252 - "GET /static/js/login.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63251 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63286 - "POST /login HTTP/1.1" 200 OK
INFO:     127.0.0.1:63286 - "GET /home HTTP/1.1" 200 OK
INFO:     127.0.0.1:63286 - "GET /static/css/styles.css HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:63286 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63288 - "GET /pending_orders HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:63288 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63322 - "GET /orders/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63322 - "GET /static/js/received_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63322 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63419 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63420 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /orders/api/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /orders/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /static/js/audit_trail.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63420 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63322 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63419 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63419 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63420 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63420 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63419 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63419 - "GET /maintenance HTTP/1.1" 200 OK
INFO:     127.0.0.1:63419 - "GET /lookups/users HTTP/1.1" 200 OK
INFO:     127.0.0.1:63322 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63416 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:63419 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:63455 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63456 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:63456 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63494 - "GET /orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63494 - "GET /static/js/pending_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63494 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63495 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63501 - "GET /static/js/components/receive_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63500 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63501 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /orders/api/orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /orders/new HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /static/js/new_order_main.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:63497 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63507 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /orders/new HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /static/js/new_order_main.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:63513 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63515 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63528 - "PUT /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63528 - "PUT /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /orders/new HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /static/js/new_order_main.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63533 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63534 - "POST /orders/generate_pdf HTTP/1.1" 200 OK
INFO:     127.0.0.1:63534 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63549 - "POST /orders/ HTTP/1.1" 200 OK
INFO:     127.0.0.1:63549 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63552 - "GET /orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63552 - "GET /static/js/pending_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63552 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63553 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63555 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63558 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63559 - "GET /static/js/components/receive_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63558 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63555 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63555 - "GET /orders/api/orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63561 - "GET /orders/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63561 - "GET /static/js/received_orders.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63561 - "GET /static/js/components/expand_line_items.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63563 - "GET /static/js/components/attachment_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63566 - "GET /static/js/components/order_note_modal.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63567 - "GET /static/js/components/shared_filters.js HTTP/1.1" 200 OK
INFO:     127.0.0.1:63567 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63567 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63567 - "GET /orders/api/received_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63567 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63569 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63590 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63603 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63611 - "GET /orders/new HTTP/1.1" 200 OK
INFO:     127.0.0.1:63611 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:63611 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63611 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63611 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:63611 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:63611 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63613 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:63613 - "GET /static/css/style.css HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:63615 - "GET /static/js/login.js HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:63615 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63613 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63629 - "POST /login HTTP/1.1" 200 OK
INFO:     127.0.0.1:63629 - "GET /home HTTP/1.1" 200 OK
INFO:     127.0.0.1:63629 - "GET /static/css/styles.css HTTP/1.1" 404 Not Found
INFO:     127.0.0.1:63629 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63631 - "GET /maintenance HTTP/1.1" 200 OK
INFO:     127.0.0.1:63631 - "GET /lookups/users HTTP/1.1" 200 OK
INFO:     127.0.0.1:63631 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63635 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:63631 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63638 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:63640 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63641 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:63643 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63677 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63677 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63684 - "GET /home HTTP/1.1" 200 OK
INFO:     127.0.0.1:63684 - "GET /static/css/style.css HTTP/1.1" 200 OK
INFO:     127.0.0.1:63684 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63692 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63696 - "GET / HTTP/1.1" 200 OK
INFO:     127.0.0.1:63696 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63698 - "POST /login HTTP/1.1" 200 OK
INFO:     127.0.0.1:63698 - "GET /home HTTP/1.1" 200 OK
INFO:     127.0.0.1:63698 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63700 - "GET /orders/new HTTP/1.1" 200 OK
INFO:     127.0.0.1:63700 - "GET /lookups/business_details HTTP/1.1" 200 OK
INFO:     127.0.0.1:63700 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63700 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63700 - "GET /lookups/items HTTP/1.1" 200 OK
INFO:     127.0.0.1:63700 - "GET /lookups/projects HTTP/1.1" 200 OK
INFO:     127.0.0.1:63700 - "GET /lookups/settings HTTP/1.1" 200 OK
INFO:     127.0.0.1:63702 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63704 - "GET /orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63704 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63706 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63706 - "GET /orders/api/orders/pending_orders HTTP/1.1" 200 OK
INFO:     127.0.0.1:63706 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63722 - "GET /orders/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63722 - "GET /lookups/suppliers HTTP/1.1" 200 OK
INFO:     127.0.0.1:63722 - "GET /lookups/requesters HTTP/1.1" 200 OK
INFO:     127.0.0.1:63724 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
INFO:     127.0.0.1:63722 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63727 - "GET /orders/api/audit_trail HTTP/1.1" 200 OK
INFO:     127.0.0.1:63729 - "GET /favicon.ico HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        self.scope, self.receive, self.send
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/sessions.py", line 85, in __call__
    await self.app(scope, receive, send_wrapper)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/fastapi/routing.py", line 338, in app
    response = actual_response_class(content, **response_args)
  File "/Users/stevencohen/Projects/universal_recycling/orders_project/venv/lib/python3.13/site-packages/starlette/responses.py", line 313, in __init__
    media_type = guess_type(filename or path)[0] or "text/plain"
                 ~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 324, in guess_type
    return _db.guess_type(url, strict)
           ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.13/3.13.3/Frameworks/Python.framework/Versions/3.13/lib/python3.13/mimetypes.py", line 125, in guess_type
    url = os.fspath(url)
TypeError: expected str, bytes or os.PathLike object, not list
